package.json

```diff
@@ -59,6 +59,7 @@
     "@stoplight/path": "^1.2.0",
     "@stoplight/types": "^11.0.0",
     "@stoplight/yaml": "^3.1.0",
+    "@types/node-fetch": "^2.5.0",
```

Comment:
Could be a dev dep.

Reply:
thanks @P0lip, I'll look into adding the test